scripts/submit_finetune_job.py

```diff
@@ -132,9 +143,13 @@ def parse_args(args):
 
     new_arguments = override_and_reconstruct_command(d1['tasks'][0]['arguments'][0], train_config, unknown_args)
     
-    # place --num_processes with args.num_gpus
+    # place --num_processes with args.num_gpus * args.num_nodes
     # will be --num_processes {N} before
-    new_arguments = re.sub(r'--num_processes \d+', f'--num_processes {args.num_gpus}', new_arguments)
+    new_arguments = re.sub(r'--num_processes \d+', f'--num_processes {args.num_gpus * args.num_nodes}', new_arguments)
```

Comment:
oh, for multinode num processes is the total processes across all machines? Thats a bit odd to me.

Reply:
I think so -- that's how I've been doing it at least and it's worked; this also follows the old og default_finetune_multinode.yaml